backend/planty/dashboard/views.py

```diff
@@ -267,6 +285,15 @@ def post(self, request: Request):
         action = serializer.validated_data['action']
         event_date: datetime = serializer.validated_data['event_date']
 
+        # if there is no time in the date add the current (time is needed for notifier)
+        if event_date.time() == time():
```

Comment:
without the comment it is slightly confusing, I expect event_date.time() == None or smth but ok